bot/cogs/git.py

```diff
@@ -131,6 +131,30 @@ async def init(self, ctx):
             )
         )
         await ctx.send(msg)
+    
+    @git.command(help='Prints info on the git pull command')
+    async def pull(self, ctx):
+        msg = ManMessage(
+            name='git-pull'
+            synopsis='git pull [options] [<repository> [<refspec>...]]',
+            options=[
+                ('-q, --quiet', 'This is passed to both underlying git-fetch to squelch reporting of during transfer, and underlying git-merge to squelch output during merging.'),
+                
+                ('-v, --verbose', 'Pass --verbose to git-fetch and git-merge.'),
+                
+                ('-r, --rebase[=false|true|preserve|interactive]', 'rebase the current branch on top of the upstream branch after fetching. If there is a remote-tracking branch corresponding to the upstream branch and the upstream branch was rebased since last fetched, the rebase uses that information to avoid rebasing non-local changes. First value is default'),
+                
+                ('--[no-]recurse-submodules[=yes|on-demand|no]', 'This option controls if new commits of all populated submodules should be fetched and updated. First value is default')
+            ],
```

Comment:
Please add a description of `<repository>` and `<refspec>`. Don't copy this directly from the official docs, try to use clear and simple language for beginners. usually `<repository>` is the name of the remote, e.g. origin and `<refspec>` the branch name.